modules/clients/pom.xml

```diff
@@ -126,6 +126,13 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
```

Comment:
I don't see usage of new dependency at all.

Reply:
Used by startRemoteGrid